packages/wrangler/src/environment-variables/factory.ts

```diff
@@ -1,5 +1,57 @@
 import { UserError } from "../errors";
 
+/**
+ * Environment variables supported by Wrangler for configuration and authentication.
+ *
+ * ## Authentication & API Configuration
+ *
+ * - `CLOUDFLARE_ACCOUNT_ID` - Overrides the account ID for API requests. Can also be set in wrangler.toml.
+ * - `CLOUDFLARE_API_TOKEN` - API token for authentication. Preferred over API key + email.
+ * - `CLOUDFLARE_API_KEY` - Legacy API key for authentication. Requires CLOUDFLARE_EMAIL.
+ * - `CLOUDFLARE_EMAIL` - Email address for API key authentication.
+ * - `CLOUDFLARE_API_BASE_URL` - Custom API base URL. Defaults to https://api.cloudflare.com/client/v4
+ * - `CLOUDFLARE_COMPLIANCE_REGION` - Set to "fedramp_high" for FedRAMP High compliance region.
```

Comment:
```suggestion
 * - `CLOUDFLARE_COMPLIANCE_REGION` - Set to "fedramp_high" for FedRAMP High compliance region. This will update the API/AUTH URLs used to make requests to Cloudflare.
```